get OSS build to pass, add VS Studio Code support

gmarzot opened the pull request:
passed tests
![image](https://github.com/user-attachments/assets/eac68f28-8256-4a60-997f-2ed2344f6546)

afrind commented on CMakeLists.txt:
```diff
@@ -51,8 +51,12 @@ find_package(folly REQUIRED)
 find_package(proxygen REQUIRED)
-# We can NOT use Glog latest version, is NOT compatible with moxygen
-find_package(glog REQUIRED)
+# We can NOT use Glog latest version, is NOT compatible with moxygen - XXX is this comment still relevant
+# find_package(glog REQUIRED) - XXX this fails OSS build
```
Can you wrap this to 80 chars?  This is our convention.

afrind replied:
@lnicco asks:

I guess I don't fully understand why for the author find_package would fail but pkg_check_modules passes.

gmarzot commented:
Hi Alan and Luca, I updated the PR based on the review. I removed those
comments altogether as the latest OS installed libgoogle-glog-dev seems to
do the trick even without any CMakeLists.txt reference. To answer Luca's
question, the find_package was failing because there were no .cmake files
being found.. which appears common for those things not built under the FB
deps manifests. The latest gihub workflows still pass so it looks good to
go from here. lmk if you see anything else. G

On Tue, Oct 22, 2024 at 9:15 PM afrind ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In CMakeLists.txt
> <https://github.com/facebookexperimental/moxygen/pull/4#discussion_r1811620926>
> :
>
> > +# We can NOT use Glog latest version, is NOT compatible with moxygen - XXX is this comment still relevant
> +# find_package(glog REQUIRED) - XXX this fails OSS build
>
> Can you wrap this to 80 chars? This is our convention.
> ------------------------------
>
> In CMakeLists.txt
> <https://github.com/facebookexperimental/moxygen/pull/4#discussion_r1811621118>
> :
>
> > +# We can NOT use Glog latest version, is NOT compatible with moxygen - XXX is this comment still relevant
> +# find_package(glog REQUIRED) - XXX this fails OSS build
>
> @lnicco <https://github.com/lnicco> asks:
>
> I guess I don't fully understand why for the author find_package would
> fail but pkg_check_modules passes.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/facebookexperimental/moxygen/pull/4#pullrequestreview-2386671275>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB7T3IAQIJ3HJ3K5WCLHMPLZ432C3AVCNFSM6AAAAABQHVQ636VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDGOBWGY3TCMRXGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
